### DISCUSSION TITLE
Data not showing when data-domain="playground.data.io,dev.data.io"

### DISCUSSION THREAD

**Discussion by andrejj** [plausible/analytics#1047] [Self-Hosted Support] [answered]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report
**Describe the bug**
[Sending data to multuple data domains (dahsboards) does not work.](https://plausible.io/docs/plausible-script#can-i-send-stats-to-multiple-dashboards-at-the-same-time)
Sending to one data domain only, does work.

**Expected behavior**
Given I have 2 dashboards setup with these domains: `playground.data.io` and `dev.data.io`
And self hosted plausible available on: `https://plausible.data.io`
And the following snippet in my html:

```html
<script async defer data-domain="playground.data.io,dev.data.io" src="https://plausible.data.io/js/plausible.js"></script
<script>window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }</script>
 ```

**Payload from gogle chrome network tab**

```json
{"n":"pageview","u":"https://playground.data.io/checkouts/af59a33d#/","d":"playground.data.io,dev.data.io","r":null,"w":1848}

```

**Environment:**
 - Self Hosted Plausible
 - Browser Chrome
 - data.io is not our real domain
 - I am sending custom events too

**Comment by metmarkosaric** [comment, marked as answer]
Hi @andrejj! The "sending data to multiple dashboards" is a new feature recently introduced to the cloud version. Bug fixes and new features are released to the cloud version several times per week. Features are battle-tested in the cloud which allows us to fix any bugs before the general self-hosted release. Every six months we combine all the changes into a new self-hosted release. So this feature will be available on self-hosted in the next release.

**Reply by andrejj** [reply]
> Every six months we combine all the changes into a new self-hosted release.

Thank you, I searched for a release schedule, but did not find any dates. Do you have an approximate date for when the release is planned?

Some context:
We have an app that multiple organizations use.
The needs are to collect data collectively (internal overview), per organization, cross organization (when orgs collaborate, they both receive data)

We opted for self hosted, because we want access to raw data.

**Reply by metmarkosaric** [reply]
You can see the releases here https://github.com/plausible/analytics/releases

The last one was in April this year so the next one should be somewhere by the end of the year

**Reply by andrejj** [reply]
Well unfortunately that is a too long wait for us, we're willing to take the risk and try with non released builds if they are available.
What if I used the [stable docker image](https://hub.docker.com/layers/plausible/analytics/stable/images/sha256-94c5014a31ad6b59da9a7ebe0c4a283c8b91e87eb077a6e25831559f63da51af?context=explore), does that one represent your cloud deployment?

**Reply by metmarkosaric** [reply]
Feel free to build your own docker image from source. The master branch includes what we run in the cloud. It changes daily, is not supported as self-hosted but if you're happy to manage it all for yourself and have the tech know-how it should work.
